mesa/visualization/templates/js/NetworkModule_d3.js

```diff
@@ -24,6 +24,24 @@ var NetworkModule = function(svg_width, svg_height) {
             g.attr("transform", d3.event.transform);
         }));
 
+    function addMarkers(element) {
+        element
+        .enter().append("svg:marker")
+        .attr("id", String)
+        .attr("viewBox", "0 -5 10 10")
+        .attr("refX", 15)
+        .attr("refY", 0.5)
+        .attr("orient", "auto")
+        .append("svg:path")
+        .attr("d", "M0,-5L10,0L0,5");
+    }
+
+    var marker_sizes = ["end", "endlarge", "endxl"];
+    for (let index = 0; index < marker_sizes.length; index++) {
```

Comment:
Use `for ... of`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
Updated to use for ... of